docs/walrus-sites/tutorial-publish.md

```diff
@@ -18,32 +18,44 @@ Since we have placed the `walrus` binary and configuration in their default loca
 the `./examples/snake` site is as simple as calling the publishing command:
 
 ``` sh
-./target/release/site-builder --config site-builder/assets/builder-example.yaml publish ./examples/snake
+./target/release/site-builder publish ./examples/snake --epochs 100
 ```
 
-The output should look like the following:
+``` admonish tip
+Currently on Walrus testnet, the duration of an epoch is 1 day. If you want your site to stay up
+longer, specify the number of epochs with the `--epochs` flag!
+```
+
+```admonish note
+The site builder will look for the default configuration file `sites-config.yaml` in the
+`./walrus-sites` directory. In case you are calling the `site-builder` command from a different
+location, use the `--config` flag to specify the path to the configuration file.
```

Comment:
Suggestion (for the future): Also look in `~/.config/walrus` for the config file as we do for `walrus`.

Reply:
yes, this has been in the back of my mind for a while. Added to the issues